packages/cf-style-provider/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I could test [yarn support|https://github.com/lerna/lerna/releases/tag/v2.0.0-beta.38]. It allows install with yarn and nothing else, this is exactly what we need.